ci/common/prep-init-env.sh

```diff
@@ -19,21 +19,32 @@ logger "Check versions..."
 gcc --version
 g++ --version
 
-# FIX Added to deal with Anancoda SSL verification issues during conda builds
-conda config --set ssl_verify False
+if [ "${CUDA:0:2}" == '10' ]; then
+  # CUDA 10 release
+  CUDA_REL=${CUDA:0:4}
+fi
 
 ################################################################################
 # BUILD - Conda package builds 
 ################################################################################
 
-CUDA_REL=${CUDA:0:3}
-if [ "${CUDA:0:2}" == '10' ]; then
-  # CUDA 10 release
-  CUDA_REL=${CUDA:0:4}
+# FIX Added to deal with Anancoda SSL verification issues during conda builds
+conda config --set ssl_verify False
+
+logger "Activate anaconda environment..."
```

Comment:
Since `gdf` anaconda env was not activated, we were testing in gpuCI with python 3.7